runtime/gc_verbose_handler_realtime/VerboseHandlerOutputRealtime.cpp

```diff
@@ -173,22 +175,24 @@ MM_VerboseHandlerOutputRealtime::getCycleType(UDATA type)
 	return cycleType;
 }
 
+void
+MM_VerboseHandlerOutputRealtime::outputInitializedInnerStanza(MM_EnvironmentBase *env, MM_VerboseBuffer *buffer){
+	MM_GCExtensionsBase *extension = env->getExtensions();
+	buffer->formatAndOutput(env, 1, "<metronome>");	
+	buffer->formatAndOutput(env, 2, "<attribute name=\"beatsPerMeasure\" value=\"%zu\" />", extension->beatMicro);
+	buffer->formatAndOutput(env, 2, "<attribute name=\"timeInterval\" value=\"%zu\" />", extension->timeWindowMicro);
+	buffer->formatAndOutput(env, 2, "<attribute name=\"targetUtilization\" value=\"%zu\" />", extension->targetUtilizationPercentage);
+	buffer->formatAndOutput(env, 2, "<attribute name=\"trigger\" value=\"0x%zx\" />", extension->gcTrigger);
+	buffer->formatAndOutput(env, 2, "<attribute name=\"headRoom\" value=\"0x%zx\" />", extension->headRoom);
+	buffer->formatAndOutput(env, 1, "</metronome>");
+}
 void
 MM_VerboseHandlerOutputRealtime::handleInitializedInnerStanzas(J9HookInterface** hook, UDATA eventNum, void* eventData)
 {
 	MM_InitializedEvent* event = (MM_InitializedEvent*)eventData;
-	MM_VerboseWriterChain* writer = _manager->getWriterChain();
 	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
-
 	handleInitializedRegion(hook, eventNum, eventData);
```

Comment:
Please look into if this needs to be changed to use buffer. I think it does like `outputInitializedInnerStanza` is